### PR TITLE
Release responsible schedule for v1.79 until v1.87

### PR DESCRIPTION
**Pull request by rfranzke**
**How to categorize this PR?**

/area delivery
/kind enhancement

**What this PR does / why we need it**:
Extends the docs with a proposal for the release responsible schedule for v1.79 until v1.87

**Release note**:

```other operator
NONE
```

### CODE REVIEW

File: docs/development/process.md [hunk 1]
```diff
@@ -21,14 +22,18 @@ Hotfixes are usually maintained for the latest three minor releases, though, the
...
-v1.72   | Week 21-22  | May 22, 2023           | June 4, 2023       | [@ary1992](https://github.com/ary1992)           |
-v1.73   | Week 23-24  | June 5, 2023           | June 18, 2023      | [@timuthy](https://github.com/timuthy)           |
-v1.74   | Week 25-26  | June 19, 2023          | July 2, 2023       | [@oliver-goetz](https://github.com/oliver-goetz) |
-v1.75   | Week 27-28  | July 3, 2023           | July 16, 2023      | [@rfranzke](https://github.com/rfranzke)         |
 v1.76   | Week 29-30  | July 17, 2023          | July 30, 2023      | [@plkokanov](https://github.com/plkokanov)       |
 v1.77   | Week 31-32  | July 31, 2023          | August 13, 2023    | [@ialidzhikov](https://github.com/ialidzhikov)   |
 v1.78   | Week 33-34  | August 14, 2023        | August 27, 2023    | [@acumino](https://github.com/acumino)           |
+v1.79   | Week 35-36  | August 28, 2023        | September 10, 2023 | [@ScheererJ](https://github.com/ScheererJ)       |
+v1.80   | Week 37-38  | September 11, 2023     | September 24, 2023 | [@shafeeqes](https://github.com/shafeeqes)       |
```
Comment (ScheererJ) [inline docs/development/process.md]:
```suggestion
v1.79   | Week 35-36  | August 28, 2023        | September 10, 2023 | [@shafeeqes](https://github.com/shafeeqes)       |
v1.80   | Week 37-38  | September 11, 2023     | September 24, 2023 | [@ScheererJ](https://github.com/ScheererJ)       |
```

**Review by ScheererJ — CHANGES_REQUESTED** [review]

**Review by ScheererJ — COMMENTED** [review]
/lgtm

**Review by timuthy — COMMENTED** [review]
/lgtm

**Comment by rfranzke** [comment]
Feedback missing from
/cc @plkokanov @ialidzhikov

**Review by shafeeqes — COMMENTED** [review]
/lgtm

**Review by acumino — COMMENTED** [review]
/lgtm

**Review by ialidzhikov — APPROVED** [review]

**Pull request merged**